scripts/create-release-branch.js

```diff
@@ -17,7 +17,7 @@ const releaseBranchName = `release/v${nextVersion}`;
 // Check if release branch has already been created
 const isRemoteReleaseBranchExist = shell
   .exec(
-    `git ls-remote --heads git@github.com:forcedotcom/salesforcedx-vscode.git ${releaseBranchName}`,
+    `git ls-remote --heads origin ${releaseBranchName}`,
```

Comment:
Give the `origin` remote alias instead since that has all the authentication needed, important for the action and still works locally.